### PR TITLE
Feanil/pally

### PR DESCRIPTION
**Pull request by feanil**

### CODE REVIEW

**Comment by feanil** [comment]
@cptvitamin Is it valuable to install just the command line pally tool to devstack?  If not I can close this PR.

**Comment by cptvitamin** [comment]
@feanil, absolutely.  I'll start to write some documentation on how developers can use pa11y on the command line and even expand it to include support on installing pa11y webservice and dashboard locally.  Thanks for this work.

**Comment by feanil** [comment]
@e0d @fredsmith @maxrothman  Please review.  It's a pretty small delta but kind of a big logical feature.

File: playbooks/roles/edxapp/tasks/deploy.yml [hunk 1]
```diff
@@ -126,6 +126,12 @@
   sudo_user: "{{ edxapp_user }}"
   environment: "{{ edxapp_environment }}"
 
+# Install Devstack tooling
```
Comment (maxrothman) [inline playbooks/roles/edxapp/tasks/deploy.yml:129]: There's a [module for npm](http://docs.ansible.com/npm_module.html). Why not use that?

File: playbooks/roles/edxapp/tasks/main.yml [hunk 2]
```diff
@@ -55,6 +55,10 @@
 - name: install system packages on which LMS and CMS rely
   apt: pkg={{','.join(edxapp_debian_pkgs)}} state=present update_cache=yes
 
+- name: install dev system packages on which LMS and CMS rely
```
Comment (maxrothman) [inline playbooks/roles/edxapp/tasks/main.yml:58]: Are you sure you need the join here?

**Pull request closed without merging**